### PR TITLE
Ansible-ize the CI test scripts

### PR DESCRIPTION
**Pull request by dry923**
Converted the test_*.sh scripts into roles. With the addition of reconciliation we are able to boil down most test scripts to a few lines of unique variable definitions and reuse roles for all the work. The two exceptions being ycsb and pgbench. Since they do not have reconciliation enabled yet they have their own specific roles. Additionally, FSdrift will be added once PR #211 is completed.

### CODE REVIEW

**Comment by dry923** [comment]
To fix the CI issues being seen on minishift this will need PR #215

File: tests/group_vars/all.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+namespace: my-ripsaw
+ripsaw_dir: /home/dry/working/ripsaw
```
Comment (dustinblack) [inline tests/group_vars/all.yml]: Does this need to be changed?

Comment (dry923) [reply tests/group_vars/all.yml]: It should be doc'd most likely. That dir gets overwritten with the current working directory when run via the test.sh script.

Comment (dustinblack) [reply tests/group_vars/all.yml]: Are you saying there are docs updated somewhere in this PR that I'm not seeing, or are you acknowledging that docs need to be done? Stupid english--

Comment (dry923) [reply tests/group_vars/all.yml]: Acknowledging that docs need to be done for this change.

Comment (dry923) [reply tests/group_vars/all.yml]: @dustinblack please check out the updated docs

**Comment by jtaleric** [comment]
@dry923 we need a rebase it seems?

**Comment by dry923** [comment]
Closing the PR for now. It may be revisited later

**Pull request closed without merging**